upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -162,6 +162,7 @@ spec:
         - -restartDnsmasq=true
         - --
         - -k
+        - -o
```

Comment:
The flag does not. `dnsmasq` is run directly (not through `dnsmasq-nanny`), and has a fairly different configuration. I can't say for certain sure it's needed, but it probably is.

Would you like me to add the `-o` flag there as well?